pkgs/applications/office/elementary-planner/default.nix

```diff
@@ -58,6 +60,8 @@ stdenv.mkDerivation rec {
     sqlite
     webkitgtk
     libgdata # required by some dependency transitively
```

Comment:
You don't know which exactly?

Reply:
I'm not sure what's the rationale for this comment, as a matter of fact `libgdata` is listed as a direct dependency in [`meson.build`](https://github.com/alainm23/planner/blob/master/meson.build).